tests/helpers/perf_run_helper.rb

```diff
@@ -683,10 +683,17 @@ def copy_system_logs(host)
                  "unknown_or_dev_job"
                end
 
-    archive_name = "#{job_name}__#{ENV['BUILD_ID']}__#{@gplt_timestamp}__system_logs.tgz"
     puppet_logdir = File.dirname on(host, puppet("config", "print", "logdir")).stdout.strip
-    archive_file_from(host, puppet_logdir,
-                      {}, @archive_root, archive_name)
+    logdir_root = puppet_logdir.split(File::SEPARATOR).select { |s| s.length >= 1 }.shift
+
+    dest = File.join(@archive_root, host, File.dirname(puppet_logdir))
+    tar_root = File.join(@archive_root, host, logdir_root)
```

Comment:
so... but why.  you are grabbing the log dir, which may I assume be in different places on different OS's... and you are naming the tar file after the top level dir...  seems like you should just call it .../puppet_log_dir.tgz

Reply:
This is the tree that is copied over from the SUT, so this is required to delete the tree afterwards.  Your point is well taken that the name of the archive should not use this value.  I have updated it below to give it the name `puppet_logdir.tgz`.  Thanks.

Reply:
yeah on the name change... that was my main thing.  I actually missed the use in the rm_rf.  Which gives me a new point... can we/should we tar it up on the SUT instead.  That would remove the need to clean up.  Another PRO would be less data moving (faster) meaning less of a chance of it doing what we saw with the soak tests and the atop log (failed due to the size of the data in transit).  The CON is the SUT has to have the space for the tar file to be created...

Reply:
@RandellP this PR has been retooled to archive the directory on the SUT and then copy the archive only.  This is basically a complete rewrite, so this PR need a complete re-review.